packages/application-shell/src/components/quick-access/butler/butler.js

```diff
@@ -430,6 +434,9 @@ export default class Butler extends React.Component {
             <input
               id="quick-access-search-input"
               ref={this.searchInputRef}
+              placeholder={this.props.intl.formatMessage(
```

Comment:
I tried for some minutes to have `FormattedMessage` work with the new FaaC API but failed as it returns a react element which the placeholder can not render.

Reply:
You mean like

```js
<FormattedMessage {...messages.inputPlacehoder}>
  {label => (
    <input placeholder={label} />
  )}
</FormattedMessage>
```

Reply:
Ah maybe that would work. How important is this for you? You'd request the change here?

Reply:
The label should be a string actually. Let me double check

Reply:
I just tried and it works as expected. Feel free to use that or not

Reply:
<img width="1440" alt="screenshot 2019-02-04 at 15 06 55" src="https://user-images.githubusercontent.com/1110551/52213140-8ac2cc80-288e-11e9-8680-869e5e3c9376.png">

Reply:
Ah, got it wrong the other way around. Thanks! I'll keep it this way however. I actually like this more as it's a bit less noisy and easy to read at the cost of using a HoC.